src/ActionList/Selection.tsx

```diff
@@ -34,33 +35,69 @@ export const Selection: React.FC<React.PropsWithChildren<SelectionProps>> = ({se
 
   /**
    * selectionVariant is multiple
-   * we use a svg instead of an input because there should not
+   * we use a styled div instead of an input because there should not
    * be an interactive element inside an option
-   * svg copied from primer/css
    */
+  const checkmarkIn = {
+    from: {
+      clipPath: 'inset(var(--base-size-16, 16px) 0 0 0)',
+    },
+
+    to: {
+      clipPath: 'inset(0 0 0 0)',
+    },
+  }
+
+  const checkmarkOut = {
+    from: {
+      clipPath: 'inset(0 0 0 0)',
+    },
+
+    to: {
+      clipPath: 'inset(var(--base-size-16, 16px) 0 0 0)',
+    },
+  }
+
   return (
-    <LeadingVisualContainer
-      sx={{
-        rect: {
-          fill: selected ? 'accent.fg' : 'canvas.default',
-          stroke: selected ? 'accent.fg' : 'border.default',
-          shapeRendering: 'auto', // this is a workaround to override global style in github/github, see primer/react#1666
-        },
-        path: {
-          fill: 'fg.onEmphasis',
-          boxShadow: 'shadow.small',
-          opacity: selected ? 1 : 0,
-        },
-      }}
-    >
-      <svg width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg" aria-hidden="true">
-        <rect x="2" y="2" width="12" height="12" rx="4"></rect>
-        <path
-          fillRule="evenodd"
-          strokeWidth="0"
-          d="M4.03231 8.69862C3.84775 8.20646 4.49385 7.77554 4.95539 7.77554C5.41693 7.77554 6.80154 9.85246 6.80154 9.85246C6.80154 9.85246 10.2631 4.314 10.4938 4.08323C10.7246 3.85246 11.8785 4.08323 11.4169 5.00631C11.0081 5.82388 7.26308 11.4678 7.26308 11.4678C7.26308 11.4678 6.80154 12.1602 6.34 11.4678C5.87846 10.7755 4.21687 9.19077 4.03231 8.69862Z"
-        />
-      </svg>
+    <LeadingVisualContainer>
+      <Box
+        sx={{
+          borderColor: selected ? 'accent.fg' : 'neutral.emphasis',
+          borderStyle: 'solid',
+          borderWidth: '1',
+          borderRadius: '1',
+          cursor: 'pointer',
+          display: 'grid',
+          height: 'var(--base-size-16, 16px)',
+          margin: '0',
+          placeContent: 'center',
+          width: 'var(--base-size-16, 16px)',
+          backgroundColor: selected ? 'accent.fg' : 'canvas.default',
+          transition: selected
```

Comment:
@mperrotti I could use some eyes on the transitions. I know something is off about it but I can't quite find it, and I think everything is backwards here compared to a `checked` state since I'm using selected.. but fresh eyes on it will probably reveal the issue.